web/admin/login.go

```diff
@@ -0,0 +1,54 @@
+package admin
+
+import (
+	"net/http"
+)
+
+type adminData struct {
+	Admin    string `yaml:"admin"`
+	Password string `yaml:"password"`
+}
+
+// Login logins admin with admin name and password.
+func (ar *Router) Login() http.HandlerFunc {
+	type loginData struct {
+		Email    string `json:"email,omitempty" validate:"required,gte=6,lte=130"`
```

Comment:
are you sure these validation rules are necessary on login? Wouldn't it be enough to just make sure these credentials match the ones in the db?

Reply:
Sure, they are not strictly necessary. I can get rid of them if needed.